docs/simple_examples.rst

```diff
@@ -0,0 +1,148 @@
+******************************
+Simple scintillometry examples
+******************************
+
+Consider observing a pulsar using a single radio dish.
+Let's assume that the pular is stationary w.r.t. Earth,
```

Comment:
Nitpick: avoid abbreviations in documentation, so just "with respect to" (or just omit it, pulsar stationary is clear enough).

Reply:
Done